libs/remix-ui/terminal/src/lib/remix-ui-terminal.tsx

```diff
@@ -116,7 +116,7 @@ export const RemixUiTerminal = (props: RemixUiTerminalProps) => {
         }
         scriptRunnerDispatch({
           type: message.type ? message.type : 'log',
-          payload: { message: [message.value] },
```

Comment:
I don't think we should do that here.
What are the input parameters?
You could have a default behavior when provider is not present.

Reply:
Here is an example of input parameters. @yann300 
```js
{
  type: 'knownTransaction',
  value: {
    tx: {
      hash: resp.detail.hash,
      isCall: false,
      blockNumber: resp.detail.ledger_seq,
      transactionCost: resp.detail.actual_fee,
      from: resp.detail.transaction.source_address,
      to: null,
      transactionIndex: resp.detail.transaction.nonce,
    },
    receipt: { status: '0x1', contractAddress: resp.detail.contractAddress },
    resolvedData: {
      contractName: selectedContract.name,
      to: null,
      fn: '(constructor)',
      params: eventsDecoder._decodeInputParams(getinputParameters(data.dataHex.replace(data.contractBytecode, '')), txHelper.getConstructorInterface(selectedContract.abi)),
      contractAddress: resp.detail.contractAddress,
    },
    logs: logs,
  },
  provider: 'bif',
}
```